### DISCUSSION TITLE
move databases content in sqlite to postgres for heroku

### DISCUSSION THREAD

**Discussion by wallythewebdev** [strapi/strapi#7404] [Legacy Topics]
Hi Guys! 

I recently pushed my strapi local to heroku - this worked without any issues: 

In Heroku my data structure is there i.e. the names of content types etc but... there is not actual data. 

So the question is: 

How can I get my content from my local dev strapi (which is made using sqlite as the db) to my production stapi in Heroku (using postgres) 

I must admit that I had assumed this would be a widely covered topic but I have found it very hard to find an answer in the docs and have spent a large sum of time looking for answers here and stackoverflow and youtube..... 

On a slightly odd side note - Why is it recommended to use sqlite when begging with Strapi if when it comes to pushing your strapi local to a live sever all the content is missing &/or unable to be moved (Im sure I'm wrong here..... but I really cannot seem to find the info that I'm looking for) - 

Any and all help is very appreciated,
All the best, 
Wally

**Comment by derrickmehaffy** [comment]
We generally recommend SQLite locally as a starter for new users as they may not be familiar with installing a database application on their host (Mongo, MySQL, MariaDB, PostgreSQL) so it's much easier for them to start without installing a bunch of packages.

I agree on the pain point of migrating your content though, and as I suggested to you in Slack. In this case the only suggestion I have here is to dump the data into a standard format and reimport that into your production database using some kind of database client such as DBeaver or HeidiSQL. But you will encounter quite a few issues when moving from a SQL based database (SQLite, PostgreSQL, MySQL, MariaDB) to something like MongoDB as the ID type is entirely different and thus will break relations as well.

As a general recommendation I suggest you use the same database you plan to use in production, locally on your development machine to make the dump => restore process easier. Also I wouldn't recommend you build a lot of content locally and try to migrate that data into production. Instead I suggest you create content in production and use that as a development/testing platform while developing locally so you can see how things will look and perform.
